tox.ini

```diff
@@ -13,9 +13,10 @@ setenv =
     PYTHONDONTWRITEBYTECODE=1
 commands =
     pip install --upgrade pip setuptools wheel
-    pip install -e .
-    pip install -e .[tests]
-    py.test {toxinidir}/csp
+    pip install pytest-cov
+    pip install --only-binary -e .
```

Comment:
Why do we need to set `--only-binary`?

Reply:
I think we can just use regular `pip install -e ...` here

Reply:
@EnTeQuAk this is in replacement of `--user-wheel`, since we're on a particular version of `pip` now. If we don't need any binaries, then we're golden then.

Reply:
Yeah, I'm sure that we don't. The `--use-wheel` usage here was very ancient.

Mind removing `--only-binary` from this PR? Besides that I think it looks really good so I'll merge this soon after.

Reply:
@EnTeQuAk done!